### PR TITLE
MSEDO-307 / MSEDO-308: Give thumbs up and down

### PR DESCRIPTION
**Pull request by DanielFran**

### CODE REVIEW

File: src/main/webapp/app/entities/therapeutic-regime/detail/therapeutic-regime-detail.component.spec.ts [hunk 1]
```diff
@@ -46,5 +55,28 @@ describe('Component Tests', () => {
...
+          comp.manageFeedback(true);
+          tick();
+
+          // THEN
+          expect(service.manageFeedbackFromEntity(feedback)).toHaveBeenCalled();
+        })
+      ));
+    });
+*/
   });
```
Comment (andrenns) [inline src/main/webapp/app/entities/therapeutic-regime/detail/therapeutic-regime-detail.component.spec.ts:81]: does it make sense to have this code commented?

File: package.json [hunk 2]
```diff
@@ -71,55 +71,56 @@
...
-    "eslint": "7.16.0",
+    "cypress": "6.2.1",
+    "eslint": "7.17.0",
     "eslint-config-prettier": "7.1.0",
     "eslint-webpack-plugin": "2.4.1",
     "friendly-errors-webpack-plugin": "1.7.0",
     "generator-jhipster": "6.10.5",
-    "husky": "4.3.6",
+    "generator-jhipster-entity-audit": "3.3.0",
+    "husky": "4.3.7",
```
Comment (andrenns) [inline package.json:123]: Was this version update discussed with the team or required?
I don't think this updates should be done without the knowledge of the team

**Comment by andrenns** [comment]
> SonarCloud Quality Gate failed.
> 
> [Bug](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=BUG)
> [Vulnerability](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=VULNERABILITY)
> [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=SECURITY_HOTSPOT)
> [Code Smell](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=CODE_SMELL) [1 Code Smell](https://sonarcloud.io/project/issues?id=MSE-2019-2021_support_care&pullRequest=83&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=MSE-2019-2021_support_care&pullRequest=83&metric=new_coverage&view=list) [60.6% Coverage](https://sonarcloud.io/component_measures?id=MSE-2019-2021_support_care&pullRequest=83&metric=new_coverage&view=list)
> [](https://sonarcloud.io/component_measures?id=MSE-2019-2021_support_care&pullRequest=83&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=MSE-2019-2021_support_care&pullRequest=83&metric=new_duplicated_lines_density&view=list)

coverage should be at least 80%

File: src/main/webapp/app/entities/therapeutic-regime/detail/therapeutic-regime-detail.component.spec.ts (hunk 1 above)
Comment (DanielFran) [reply src/main/webapp/app/entities/therapeutic-regime/detail/therapeutic-regime-detail.component.spec.ts]: Removed

File: package.json (hunk 2 above)
Comment (DanielFran) [reply package.json:123]: This was to update to final version of @angular-builders

Comment (andrenns) [reply package.json:123]: Was this discussed with the team?

**Pull request merged**